backend/shared/src/create-notification.ts

```diff
@@ -1266,12 +1266,12 @@ export const createContractResolvedNotifications = async (
         profitRank && totalShareholders && betterThan > 0
           ? `, outperforming ${betterThan} other${betterThan > 1 ? 's' : ''}!`
           : '.'
-      const profit = Math.round(userPayout - userInvestment)
+      const profit = userPayout - userInvestment
       const profitPercent = Math.round((profit / userInvestment) * 100)
-      const profitString = ` You made M${getMoneyNumber(
+      const profitString = ` You made ${formatMoney(
         profit
       )} (+${profitPercent}%)`
-      const lossString = ` You lost M${getMoneyNumber(-profit)}`
+      const lossString = ` You lost ${formatMoney(-profit)}`
```

Comment:
Seems like you could do profit >= 0 instead of profit > 0 on line 1284 to achieve what you want.

Reply:
The trouble is small losses (-0.5,0.0) and small wins (0.0, 0.5) both get treated the same if we use the rounded profit for the comparison with 0.